app/views/404.hbs

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <title>Streetmix - Page not Found</title>
+
+    <meta charset="utf-8">
+    <meta http-equiv="Content-Language" content="en">
+    <meta name="description" content="Design, remix, and share your neighborhood street – all in your browser! Add trees or bike paths, widen sidewalks or traffic lanes, learn how your decisions can impact your community.">
+
+    {{! Display at 50% on mobile devices only; should not affect desktop browsers }}
+    <meta name="viewport" content="width=device-width, initial-scale=.5, maximum-scale=.5">
```

Comment:
Let's change this to a more mobile-standard `initial-scale=1, maximum-scale=1` for this page.